app/src/Policy/ArticlePolicy.php

```diff
@@ -0,0 +1,77 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Policy;
+
+use App\Model\Entity\Article;
+use Authorization\IdentityInterface;
+
+/**
+ * Article policy
+ */
+class ArticlePolicy
+{
+    /**
+     * Check if $user can add Article
+     *
+     * @param \Authorization\IdentityInterface $user The user.
+     * @param \App\Model\Entity\Article $article current article.
+     * @return bool
+     */
+    public function canAdd(IdentityInterface $user, Article $article)
+    {
+        return true;
+    }
+
+    /**
+     * Check if $user can edit Article
+     *
+     * @param \Authorization\IdentityInterface $user The user.
+     * @param \App\Model\Entity\Article $article current article.
+     * @return bool
+     */
+    public function canEdit(IdentityInterface $user, Article $article)
+    {
+        return $this->isAuthor($user, $article);
+        // return true;
+    }
+
+    /**
+     * Check if $user can delete Article
+     *
+     * @param \Authorization\IdentityInterface $user The user.
+     * @param \App\Model\Entity\Article $article current article.
+     * @return bool
+     */
+    public function canDelete(IdentityInterface $user, Article $article)
+    {
+        return $this->isAuthor($user, $article);
+    }
+
+    /**
+     * Check if $user can view Article
+     *
+     * @param \Authorization\IdentityInterface $user The user.
+     * @param \App\Model\Entity\Article $article current article.
+     * @return bool
+     */
+    public function canView(IdentityInterface $user, Article $article)
+    {
+        return true;
+    }
+
+    /**
+     * Check if $user is article's author
+     *
+     * @param \Authorization\IdentityInterface $user The user.
+     * @param \App\Model\Entity\Article $article current article.
+     * @return bool
+     */
+    protected function isAuthor(IdentityInterface $user, Article $article): bool
+    {
+        /** @var \Authentication\IdentityInterface $authUser */
+        $authUser = $user;
+
+        return $article->user_id === $authUser->getIdentifier();
```

Comment:
```suggestion
        /** @var \App\Model\Entity\User $userEntity */
        $userEntity = $user->getOriginalData();

        return $article->user_id === $userEntity->id;
```

Please refer https://book.cakephp.org/authorization/2/en/middleware.html#identity-decorator